src/scaict_uwu/service_wiring.py

```
@@ -0,0 +1,40 @@
+    return {
+        "Config": get_config,
+        "ConfigFactory": get_config_factory,
+        "LanguageTagFactory": get_language_tag_factory,
+    }
+
+
+def get_config(service_container: ServiceContainer) -> Config:
+    return service_container.get_config_factory().get()
+
+
+def get_config_factory(service_container: ServiceContainer) -> ConfigFactory:
```

Ruff (0.8.0)

Unused function argument: `service_container`

(ARG001)